Xamarin.Essentials/Connectivity/Connectivity.android.cs

```diff
@@ -21,6 +22,20 @@ static void StartListeners()
             Platform.AppContext.RegisterReceiver(conectivityReceiver, new IntentFilter(ConnectivityManager.ConnectivityAction));
         }
 
+        static SignalStrength PlatformSignalStrength()
+        {
+            var wifiManager = Platform.WifiManager;
```

Comment:
Need to wrap this in a try/catch and need to ensure that we have the WiFi permission set else it will throw. See other example in here.

Reply:
done